net/boinc-wrapper/Makefile

```diff
@@ -0,0 +1,78 @@
+# Copyright 2022 by Vitalii Koshura
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=boinc-wrapper
+PKG_VERSION:=26018
+PKG_RELEASE:=1
+
+PKG_SOURCE_DATE:=2022-09-07
```

Comment:
Isn't this used mostly when you are cloning Git repositories? This could be removed. 

Reference: https://openwrt.org/docs/guide-developer/package-policies#versioning

Reply:
True, will remove that

Reply:
Done